packages/fury-adapter-oas3-parser/STATUS.md

```diff
@@ -106,6 +106,14 @@ Key:
 | header | ✕ |
 | cookie | ✕ |
 
+## Request Body Object
```

Comment:
Missing `| requestBody | ✕ |` link up there?

Reply:
I was going to do that once `requestBody` in Operation Object uses the request body parser.

Such as:

```diff
diff --git a/packages/fury-adapter-oas3-parser/STATUS.md b/packages/fury-adapter-oas3-parser/STATUS.md
index 7d28f98..9d7bd11 100644
--- a/packages/fury-adapter-oas3-parser/STATUS.md
+++ b/packages/fury-adapter-oas3-parser/STATUS.md
@@ -68,7 +68,7 @@ Key:
 | externalDocs | [✕](https://github.com/apiaryio/api-elements.js/issues/83) |
 | operationId | ✓ |
 | parameters | [✕](https://github.com/apiaryio/api-elements.js/issues/65) |
-| requestBody | ✕ |
+| requestBody | [~](#request-body-object) |
 | responses | [~](#responses-object) |
 | callbacks | [✕](https://github.com/apiaryio/api-elements.js/issues/74) |
 | deprecated | ✕ |
diff --git a/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOperationObject.js b/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOperationObject.js
index 24d802d..16755b4 100644
--- a/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOperationObject.js
+++ b/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOperationObject.js
@@ -13,12 +13,12 @@ const pipeParseResult = require('../../pipeParseResult');
 const parseObject = require('../parseObject');
 const parseString = require('../parseString');
 const parseResponsesObject = require('./parseResponsesObject');
+const parseRequestBodyObject = require('./parseRequestBodyObject');

 const name = 'Operation Object';
 const requiredKeys = ['responses'];
 const unsupportedKeys = [
-  'tags', 'externalDocs', 'parameters', 'requestBody',
-  'callbacks', 'deprecated', 'security',
+  'tags', 'externalDocs', 'parameters', 'callbacks', 'deprecated', 'security',
 ];
 const isUnsupportedKey = R.anyPass(R.map(hasKey, unsupportedKeys));

@@ -52,6 +52,7 @@ function parseOperationObject(context, member) {
     [hasKey('description'), parseCopy(context, name, false)],
     [hasKey('operationId'), parseString(context, name, false)],
     [hasKey('responses'), R.compose(parseResponsesObject(context), getValue)],
+    [hasKey('requestBody'), R.compose(parseRequestBodyObject(context), getValue)],

     [isUnsupportedKey, createUnsupportedMemberWarning(namespace, name)],
```

Reply:
😄